pkg/transport/generic_transport_controller.go

```diff
@@ -394,6 +443,8 @@ func (c *genericTransportController) processNextWorkItem(ctx context.Context) bo
 // to which that matters. The string is the name of the inventory object.
 type recollectProperties string
 
+type customTransformReference string // the name of the cluster-scoped object
```

Comment:
FYI, of all the items the workqueue supports, only the binding's key remains an unaliased string type, do you think that it should remain like that?

Reply:
I noticed that too. I left it as is, for the sake of not changing more than necessary. I think we can address this (and some other things) after this PR merges.